Refactored

panchami6 opened the pull request.

uditkumar01 commented:
Everything looks perfect. But if u wish u can improve your code by using de-structuring instead of using dot operator. U have used `item.name` inside some of map functions.

shivamupadhyay008 reviewed (commented):
everything is great some minor bugs in layout in search bar search icon is not align well and like button should be visible more

Akshaay9 commented:
Hey panchami,
Everything looks fine

 U can add a loader in home page while fetching data, to show that page is loading

In cart list, when the item qty is 1, instead of making the `-` button disable, u can use that button to remove item from qty(if the product qty is 1)

Thank you

AnkitKarnAK commented on src/Pages/Products.js:
```diff
@@ -1,212 +1,211 @@
+              height="auto"
+              alt={item.productName}
+            />
+            {console.log(typeof item.image)}
```
Remove this console.log, this shouldn't be needed in the main branch

AnkitKarnAK reviewed (commented):
Hey, Panchami. your code looks clean. I have added a comment on products.js file of having console.log which isn't needed in production

panchami6 commented:
> everything is great some minor bugs in layout in search bar search icon is not align well and like button should be visible more

Hi Shivam, Thanks for reviewing my code. I have fixed the search icon alignment.

panchami6 commented:
> Hey, Panchami. your code looks clean. I have added a comment on products.js file of having console.log which isn't needed in production

Hi Ankit, Thanks for reviewing my code. I have removed the console logs

panchami6 commented:
> Hey panchami,
> Everything looks fine
> 
> U can add a loader in home page while fetching data, to show that page is loading
> 
> In cart list, when the item qty is 1, instead of making the `-` button disable, u can use that button to remove item from qty(if the product qty is 1)
> 
> Thank you

Hi Akshay, I have added the loader now. Thanks for the suggestion.

panchami6 commented:
> Everything looks perfect. But if u wish u can improve your code by using de-structuring instead of using dot operator. U have used `item.name` inside some of map functions.

Hi Udit, sure will work on that. Thanks for the suggestion
